Since the package build happens in the pmaports git repo, this check doesn't give the correct result and we should disable it.

This is not an upstreamable fix as it just removes the check rather than replacing it with something more proper.

diff --git a/libuuu/gen_ver.sh b/libuuu/gen_ver.sh
index d4e8c1ba..8f416322 100755
--- a/libuuu/gen_ver.sh
+++ b/libuuu/gen_ver.sh
@@ -13,10 +13,10 @@ else
 fi
 
 # Test if we are in a repo
-if [ "$(git rev-parse --is-inside-work-tree 2>/dev/null)" = "true" ];
-then
-	#echo "In a repo"
-	# Get the version of the last commit of the repo
-	version=`git describe --tags --long`
-	echo "#define GIT_VERSION \"lib$version\"" > $file_to_write
-fi
+#if [ "$(git rev-parse --is-inside-work-tree 2>/dev/null)" = "true" ];
+#then
+#	#echo "In a repo"
+#	# Get the version of the last commit of the repo
+#	version=`git describe --tags --long`
+#	echo "#define GIT_VERSION \"lib$version\"" > $file_to_write
+#fi
